### PR TITLE
AP-4042: HMRC design amendments

### PR DESCRIPTION
**Pull request by skdejong**
## What
HMRC design amendments

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4042)

Add content to landing page in line with [designs](https://www.figma.com/file/HrVZGjvgsCYqhLt5O7UHVI/HMRC-assurance-tool?node-id=232-13&t=2LuJZe18mpDCEjCM-0). 

## Checklist

Before you ask people to review this PR:

- Tests and linters should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by colinbruce — APPROVED** [review]
Looks good to me :+1:
🏆

File: spec/system/sign_in_spec.rb [hunk 1]
```diff
@@ -17,8 +17,11 @@
     it "takes user to home page" do
       visit "/"
       click_button "Start now"
+      expect(page).to have_selector(".govuk-header__navigation-item--active")
```
Comment (jsugarman) [inline spec/system/sign_in_spec.rb:20]: i don't think we need this line as you are checking the correct link is "active" below, which is a preferred way

Comment (skdejong) [reply spec/system/sign_in_spec.rb:20]: Thanks for the tip. I have deleted this line now.

**Review by jsugarman — COMMENTED** [review]
some other comments @skdejong relating to having two "Check details" content on the page (heading/h1 and link) and how we should be distinguishing between them in the tests ideally

**Pull request merged**